addons/l10n_cl/models/account_move.py

```diff
@@ -160,6 +161,36 @@ def _l10n_cl_get_invoice_totals_for_report(self):
             tax_totals['formatted_amount_untaxed'] = formatLang(self.env, tax_totals['amount_untaxed'], currency_obj=self.currency_id)
             if tax_totals['subtotals']:
                 tax_totals['subtotals'][0]['formatted_amount'] = tax_totals['formatted_amount_untaxed']
+        values = self._l10n_cl_get_amounts()
+        withholdings = self._l10n_cl_get_withholdings()
+        if self.currency_id != self.env.ref('base.CLP'):
+            tax_totals['formatted_amount_total'] = formatLang(
+                self.env, values['total_amount'], currency_obj=self.env.ref('base.CLP'))
+            tax_totals['subtotals'][0]['amount'] = values['subtotal_amount_taxable']
+            tax_totals['subtotals'][0]['formatted_amount'] = formatLang(
+                self.env, values['subtotal_amount_taxable'], currency_obj=self.env.ref('base.CLP'))
+        list_of_tax_groups = []
+        for groups_by_subtotal in tax_totals['groups_by_subtotal']['Base imponible']:
```

Comment:
if _('Untaxed Amount') in totals['groups_by_subtotal']:

Reply:
cff2343c but didn't work